src/test/java/TasksServletTest.java

```diff
@@ -112,48 +143,109 @@ public void noTaskLists() throws IOException, ServletException {
     Mockito.when(tasksClient.listTaskLists()).thenReturn(NO_TASKLISTS);
     servlet.doGet(request, response);
     printWriter.flush();
-    List<Task> tasks = gson.fromJson(stringWriter.toString(), LIST_OF_TASKS_TYPE);
-    Assert.assertTrue(tasks.isEmpty());
+    TasksResponse tasksResponse = gson.fromJson(stringWriter.toString(), TasksResponse.class);
+    Assert.assertEquals(NO_TASKLISTS_TITLES, tasksResponse.getTaskListNames());
+    Assert.assertEquals(0, tasksResponse.getTasksToComplete());
+    Assert.assertEquals(0, tasksResponse.getTasksDueToday());
+    Assert.assertEquals(0, tasksResponse.getTasksCompletedToday());
+    Assert.assertEquals(0, tasksResponse.getTasksOverdue());
```

Comment:
You could override `Object.equals` in `TasksResponse`, and then use `Assert.assertEquals` to compare `TasksResponse` objects directly.

Reply:
This would make it a lot neater, thanks!

Reply:
`RESOLVED` https://github.com/googleinterns/step7-2020/pull/105